src/numberMask/index.js

```diff
@@ -1,20 +1,15 @@
-import * as R from 'ramda';
-
-// Removes non numeric values
-const removeNonNumeric = value => value.replace(/[^0-9.]+/g, '');
-
-const addCommasToNumber = (value = '') => value.replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');
-
+/**
+ * Given a Number, return a masked version with commas and squash to 2 fractionals
+ * @param {Number} value Number to be masked
+ * @returns String
+ */
 const numberMask = (value) => {
-  const stringValue = String(value);
-  const isNegative = stringValue.charAt(0) === '-';
+  if (!value) return '';
```

Comment:
This PR is almost 💯 

But minor thing, if `value === 0` we need to return `0` not an empty string